templates/web/fixmystreet.com/header_extra.html

```diff
@@ -2,6 +2,22 @@
 <link rel="prefetch" href="[% version('/vendor/OpenLayers.Projection.OrdnanceSurvey.js') %]">
 [% END %]
 
+[% IF bodyclass.match('mappage') AND c.config.BASE_URL == "https://www.fixmystreet.com" %]
+<script src="//www.google-analytics.com/cx/api.js?experiment=qcO31Qq5RZC3J0U9cBB2xw"></script>
+<script nonce="[% csp_nonce %]">
+    var variation = typeof cxApi !== "undefined" ? cxApi.chooseVariation() : 0,
+        docElement = document.documentElement,
+        className = docElement.className;
+    docElement.className = className + ' banner-position-variant-' + variation;
+</script>
+<style>
+    html .banner-position-variant-1 { display: none !important; }
+    html.banner-position-variant-1 .banner-position-variant-0 { display: none !important; }
+    html.banner-position-variant-1 .banner-position-variant-1 { display: block !important; }
+    html.banner-position-variant-1 #side .big-green-banner ~ section.full-width { margin-top: -1em; }
+</style>
+[% END %]
```

Comment:
The style needs to be outside of (at least some of) the `IF`, otherwise non-live sites will have both buttons.

Reply:
Except, if I add the `fixmystreet.com` check around `<h1 class="big-green-banner banner-position-variant-1">` in `display_location.html`, as you suggested above, then I assume I won’t need to worry about this, since the `.banner-position-variant-1` banner will only appear on fixmystreet.com?

Reply:
Oh, sorry, for the `IF` I mentioned I assumed it'd be `IF c.cobrand.moniker == 'fixmystreet'` - otherwise you've made it impossible to do any development on it ;) But yes, if it's a "live-only" `IF` then you're right, it just makes further development a bit harder (though I assume there won't be any/much).

Reply:
Haha! Yeah, I just disabled that part of the check when I needed to try it out locally. Figured we probably don’t want development installs sending hits back to our GA experiment. (Although now I think about it, maybe this is _already_ mitigated by whatever we already have in place to prevent GA pageviews/events being recorded in development. Hmmm…)